bedrock/products/templates/products/monitor/cms/includes/macros.html

```diff
@@ -0,0 +1,37 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% macro picto_icon(image) -%}
+  {# TKTK: To do: this is SVG, input should only accept SVG, this is one size #}
+  {{ srcset_image(
+    image,
+    "width-{200,400,600,800,1000,1200}",
+    class="",
+    sizes="(min-width: 1024px) 33vw', (min-width: 480px) 50vw', 100vw",
+    width="40",
+    height="40",
+    alt="TKTK fix sizing, should only be SVG",
+    loading="lazy"
+  ) }}
+{%- endmacro %}
+
+
+{% macro split_media(image) %}
+  {# TKTK: To do: available widths should include max needed size - 1360 #}
```

Comment:
Can we use the `width-1400` that we get from the pre-generated rendtion sizes [mentioned here](https://bedrock.readthedocs.io/en/latest/cms.html#using-editor-uploaded-images-in-templates)? Surely that 40px difference won't be a big visual problem?

If not, the only realistic route we have is to add 1360 to that list of pre-generations, but I'm keen to avoid that list getting longer, because it makes it for _every_ image we upload. (It's a constraint of having readonly infra in production - renditions are made just-in-time if they don't exist already, but that fails hard when the Web pods can't write to the DB or the bucket storage). Alex and I thought that 200px increments would cover most things.

Reply:
Yeah, unless there’s a specific reason why a 1400px image would not work here, I think we should try and stick to predefined values. It makes working with images on the front end a bit easier too imho, since it’s simpler to round up and avoid dealing with very specific magic numbers. I don’t think a few pixels difference is going to make much impact on performance.

Reply:
Yeah, sorry, it wasn't a feature request. I had not yet dug into the docs to figure out how to include larger sizes. 1400 is great I just didn't know what the next sizes up were.